packages/headless/doc-parser/src/interface-resolver.test.ts

```diff
@@ -698,6 +698,56 @@ describe('#resolveInterfaceMembers', () => {
     expect(result).toEqual([expected]);
   });
 
+  it(`an interface extends two interfaces,
+  both inherited interfaces have a property with the same name,
+  it only keeps one inherited instance`, () => {
+    const entry = buildMockEntryPoint();
+    const standaloneSearchBox = buildMockApiInterface({
+      name: 'StandaloneSearchBox',
+      excerptTokens: [
+        buildContentExcerptToken('interface StandaloneSearchBox extends '),
```

Comment:
This just made me realize: why is `StandaloneSearchBox` extending both `SearchBox` and `Controller` when `SearchBox` already extends `Controller`?